src/app.config.json

```diff
@@ -0,0 +1,69 @@
+{
+    "$schema": "../node_modules/@alfresco/adf-core/app.config.schema.json",
+    "ecmHost": "http://localhost",
+    "bpmHost": "http://localhost",
+    "backendHost": "http://localhost:8080",
```

Comment:
I find this naming of configuration variables rather confusing: we are naming variables `<something>Host`, yet they actually hold URLs.

Reply:
I didn't name those for ACS and APS. They are generated. I created just the one for backend service and I tried to follow the pattern.